src/functions.php

```diff
@@ -331,3 +331,14 @@ function json_encode($value, $options = 0, $depth = 512)
 
     return $json;
 }
+
+/**
+ * Wrapper for the hrtime() or microtime() functions
+ * (depending on the PHP version, one of the two is used)
+ *
+ * @return float|mixed UNIX timestamp
+ */
```

Comment:
I would suggest adding the internal annotation and also follow the convention of adding an underscore at the start of the function name... as it is done on other functions IIRC

Reply:
I don't see any of the other functions having the internal annotation nor having an underscore prefix. They are all under a namespace, so no name collisions should happen.

Reply:
> I would suggest adding the internal annotation and also follow the convention of adding an underscore at the start of the function name

No it's not?

Reply:
> so no name collisions should happen.

Yes, exactly.

Reply:
anyway not sure if this is a direction that must be followed... someone from guzzle could validate

Reply:
Only others of the functions in that file is like that, others are all not.